src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php

```diff
@@ -56,6 +56,7 @@ public function process(ContainerBuilder $container)
         foreach ($commandServices as $id => $tags) {
             $definition = $container->getDefinition($id);
             $definition->addTag($this->noPreloadTag);
+            /** @var class-string<Command> $class */
```

Comment:
This change looks unrelated and should be reverted.

Reply:
Done